wp/lib/bap_wp/src/postprocess.ml

```diff
@@ -0,0 +1,133 @@
+(***************************************************************************)
+(*                                                                         *)
+(*  Copyright (C) 2018/2019 The Charles Stark Draper Laboratory, Inc.      *)
+(*                                                                         *)
+(*  This file is provided under the license found in the LICENSE file in   *)
+(*  the top-level directory of this project.                               *)
+(*                                                                         *)
+(*  This work is funded in part by ONR/NAWC Contract N6833518C0107.  Its   *)
+(*  content does not necessarily reflect the position or policy of the US  *)
+(*  Government and no official endorsement should be inferred.             *)
+(*                                                                         *)
+(***************************************************************************)
+
+open !Core_kernel
+open Bap.Std
+
+include Self()
+
+module Expr = Z3.Expr
+module Arith = Z3.Arithmetic
+module BV = Z3.BitVector
+module Bool = Z3.Boolean
+module Array = Z3.Z3Array
+module Env = Environment
+module Constr = Constraint
+
+
+(** Implements {!Model}. *)
+
+open Core_kernel
+open Bap.Std
+
+type sign = Unsigned | Signed
+
+let string_of_sign s =
```

Comment:
Can you annotate the types for the inputs and output for the functions in this module?

Reply:
I should trash a lot of these anyway. They aren't used.

Reply:
My statement was false. They are used.